### ISSUE TITLE
Python library installation prefix

### ISSUE THREAD

**Issue by Funkensieper** [PCMDI/cmor#169]
The python library is not installed under the installation prefix given via --prefix, instead the installer tries to update the current python installation (which fails if the user doesn't have permission to do that). I discovered that you can change the python installation prefix via --with-cdat, but I would expect both the C and the python library to be installed under the same prefix. At least it would be good to mention that option in the documentation or the INSTALL file.

**Comment by dnadeau4** [comment]
@Funkensieper I set CDATPREFIX to `--prefix=${prefix}` by default and set the `PYTHONPATH` to `${prefix}`   That seems to have done the needed magic.   It will be in version `3.2.4` but if you need it now, I have attached a patch.

Just apply `patch < cmor.patch`

=====
# cmor.patch
```
--- Makefile.in	2017-05-10 09:58:38.281670126 -0700
+++ Makefile.in.new	2017-05-10 09:58:33.417693232 -0700
@@ -6,6 +6,7 @@
 USERFFLAGS=@USERFFLAGS@
 #MODULE_SUFFIX="mod"
 PREFIX=@prefix@
+PYTHONPATH=@prefix@/lib/python2.7/site-packages
 FCFLAGS=@FCFLAGS@
 PYTHONEXEC=@PYTHONEXEC@
 NO_COLOR2=\x1b[0m
--- configure	2017-05-10 10:00:30.861135367 -0700
+++ configure.new	2017-05-10 10:00:27.633150699 -0700
@@ -2264,7 +2264,7 @@
   MODFILES="cmor_users_functions.*"
 fi
 
-CDATPREFIX=" "
+CDATPREFIX="--prefix=${prefix}"
 MAKETESTPYTHON=" "
 
 # Check whether --with-cdat was given.
```

**Comment by dnadeau4** [comment]
# cmor.path (with directory creation)
```--- Makefile.in	2017-05-10 10:09:53.618462227 -0700
+++ Makefile.in.new	2017-05-10 10:09:33.794556392 -0700
@@ -6,6 +6,7 @@
 USERFFLAGS=@USERFFLAGS@
 #MODULE_SUFFIX="mod"
 PREFIX=@prefix@
+PYTHONPATH=@prefix@/lib/python2.7/site-packages
 FCFLAGS=@FCFLAGS@
 PYTHONEXEC=@PYTHONEXEC@
 NO_COLOR2=\x1b[0m
@@ -145,6 +146,7 @@
 endif
 python:
 	@echo "Building Python interface"
+	@if [ ! -d "${PYTHONPATH}" ]; then mkdir -p ${PYTHONPATH}; fi
 	@${PYTHONEXEC} setup.py install @CDATPREFIX@
 test_a_python:
 	@echo "${OK_COLOR}Testing ${TEST_NAME} ${NO_COLOR}"
--- configure	2017-05-10 10:00:30.861135367 -0700
+++ configure.new	2017-05-10 10:00:27.633150699 -0700
@@ -2264,7 +2264,7 @@
   MODFILES="cmor_users_functions.*"
 fi
 
-CDATPREFIX=" "
+CDATPREFIX="--prefix=${prefix}"
 MAKETESTPYTHON=" "
 
 # Check whether --with-cdat was given.
```

**Comment by Funkensieper** [comment]
Great, thanks!

**Comment by dnadeau4** [comment]
Solved in 3.2.4 Please test.
